Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Rifles/L24_rifle.yml

```
@@ -0,0 +1,399 @@
+      sprite: _RMC14/Objects/Weapons/Guns/Ammunition/Magazines/extendedL24.rsi
+      layers:
+      - state: base
+        map: [ "enum.GunVisualLayers.Base" ]
+      - state: mag-1
+        map: [ "enum.GunVisualLayers.Mag" ]
+    - type: Appearance
+
+- type: entity
+  parent: CMBaseCartridgeRifle
+  id: RMCCartridgeRifleL24
+  name: Cartridge (8.88x51mm)
+  description: A 8.88x51mm rifle cartridge
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - RMCCartridgeRifleL24
+  - type: CartridgeAmmo
+    proto: BulletRifle888x51mm
+
+- type: entity
+  parent: CMBaseCartridgeRifle
+  id: RMCCartridgeRifleL24AP
+  name: AP Cartridge (8.88x51mm)
+  description: A 8.88x51mm AP rifle cartridge
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - RMCCartridgeRifleL24AP
+  - type: CartridgeAmmo
+    proto: BulletRifle888x51mmAP
+
+- type: entity
+  parent: CMBaseCartridgeRifle
+  id: RMCCartridgeRifleL24HEAP
+  name: HEAP Cartridge (8.88x51mm)
+  description: A 8.88x51mm HEAP rifle cartridge
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - RMCCartridgeRifleL24HEAP
+  - type: CartridgeAmmo
+    proto: BulletRifle888x51mmHEAP
+
+- type: entity
+  parent: CMBaseCartridgeRifle
+  id: RMCCartridgeRifleL24Incendiary
+  name: Incendiary Cartridge (8.88x51mm)
+  description: A 8.88x51mm Incendiary rifle cartridge
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - RMCCartridgeRifleL24Incendiary
+  - type: CartridgeAmmo
+    proto: BulletRifle888x51mmIncendiary
+
+- type: entity
+  parent: BulletRifle10x24mm
+  id: BulletRifle888x51mmAP
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Piercing: 40
+  - type: CMArmorPiercing
+    amount: 50
+
+- type: entity
+  parent: BulletRifle10x24mm
+  id: BulletRifle888x51mm
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+       types:
+         Piercing: 40
+  - type: CMArmorPiercing
+    amount: 10
+
+- type: entity
+  parent: BulletRifle10x24mm
+  id: BulletRifle888x51mmHEAP
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+       types:
+         Piercing: 65
+  - type: CMArmorPiercing
+    amount: 50
+
+- type: entity
+  parent: BulletRifle10x24mm
+  id: BulletRifle888x51mmIncendiary
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+       types:
+         Piercing: 40
+  - type: CMArmorPiercing
+    amount: 10
+  - type: IgniteOnProjectileHit
+
+
+- type: Tag
+  id: RMCMagazineRifleL24
+
+- type: Tag
+  id: RMCMagazineRifleL24AP
+
+- type: Tag
+  id: RMCMagazineRifleL24HEAP
+
+- type: Tag
+  id: RMCMagazineRifleL24Incendiary
+
+- type: Tag
+  id: RMCMagazineRifleL24Extended
+
+- type: Tag
+  id: RMCCartridgeRifleL24
+
+- type: Tag
+  id: RMCCartridgeRifleL24AP
+
+- type: Tag
+  id: RMCCartridgeRifleL24HEAP
+
+- type: Tag
+  id: RMCCartridgeRifleL24Incendiary
+
+- type: Tag
+  id: WeaponRifleL24
```

```suggestion
  id: RMCWeaponRifleL24
```
also it seems to be unused? If so - remove it.